easybuild/easyconfigs/t/tqdm/tqdm-4.66.1-GCCcore-12.3.0.eb

```diff
@@ -16,6 +16,7 @@ checksums = ['d88e651f9db8d8551a62556d3cff9e3034274ca5d66e93197cf2490e2dcb69c7']
 
 builddependencies = [
     ('binutils', '2.40'),
+    ('hatchling', '1.18.0')
```

Comment:
Nit;
```suggestion
    ('hatchling', '1.18.0'),
```
This allows adding more if required without changing this line (you can click the accept button here to avoid having to do this manually and pushing again)